packages/flutter_tools/test/general.shard/ios/ios_workflow_test.dart

```diff
@@ -41,16 +45,25 @@ void main() {
     );
 
     expect(iosWorkflow.appliesToHostPlatform, false);
+    expect(iosWorkflow.canLaunchDevices, false);
+    expect(iosWorkflow.canListDevices, false);
   });
 
-  testWithoutContext('iOS workflow is enabled on macOS', () {
+  testWithoutContext('iOS workflow applies on macOS, no Xcode', () {
```

Comment:
The next test was the same set up. Instead, make this one check what happens when Xcode is not installed.